app/grants/models/grant_branding_routing_policy.py

```diff
@@ -33,6 +33,13 @@ class GrantBrandingRoutingPolicy(SuperModel):
         null=True
     )
     inline_css = models.TextField(default='', blank=True, help_text=_('Inline css to customize the banner fit'))
+    main_round_banner = models.ImageField(
```

Comment:
This seems like an overkill making a model change just for the main round banner !
Could we avoid doing this all together and just rely on fetching this from JSON store table

Reply:
This seemed like what the ticket was asking for, a way to upload via the admin panel. I'm not sure how this would be overkill

Reply:
So adding a new field to to table for a field meant to be just used once seems un-necessary. 
We could use [staticjsonenv](https://gitcoin.co/_administrationperftools/staticjsonenv/) table and have admin enter the uploaded image URL instead and just fetch that instead as opposed to make a new column in GrantBrandRoundingPolicy ?

Unless i'm missing something here and this field will be populated for every field in the GrantBrandRoutingPolicyTable